scripts/pre-test.sh

```diff
@@ -0,0 +1,9 @@
+NYC=./.nyc_output
```

Comment:
`nyc` handles this for us automatically, I believe.

Reply:
`cypress run` has a funny behavior where it doesn't generate code coverage while running in `headless` mode. The hacky way around this is creating the `./.nyc_output/out.json` file which is recognized by the `code-coverage` plugin that interacts with NYC. 
Basically this kick-starts the `code-coverage` plugin to generate and output.

I would leave this as is until a better solution is available.

Reply:
Oh! 

@BolajiAyodeji can you clarify if you're sending coverage for e2e tests or just unit tests?
My thought is that we don't need coverage for e2e because it either works or it doesn't but will defer to @BolajiAyodeji for guidance here.

If we are sending coverage for both e2e and unit tests then with what you have here, the coverage is going to be always overwritten.

Reply:
Yes. It would be overwritten and `code-coverage` would be generated every time the test runs.
@BolajiAyodeji please comment on this. This issue is overdue.